### PR TITLE
Split README into separate documenation pages

### PR DESCRIPTION
**Pull request by pgarrison**
# Issues addressed
* README is dense and organization is unclear. We want to write to multiple audiences who want to use this for different purposes.
* Docs include paths in tree diagrams that I didn't update when moving to the `src/` and `subpackages/` structure.
* `image_preprocessing` usage instructions say `--cores ..` but `..` is just a placeholder (#39)
* Preprocessing docs make dependence on two virtual environments more clear

# Reviewing
[See new version here](https://github.com/AllenCell/benchmarking_representations/blob/overhaul-readme/README.md)

What do you all think about moving the reproducibility and dev documentation out of the main README and instead highlighting the "educational" example(s) @ritvikvasan proposed there?

### CODE REVIEW

**Comment by ritvikvasan** [comment]
Looks much better to me!

**Review by vianamp — APPROVED** [review]
I like this. As mentioned in Slack, replace the "movies" in the figure with single cell images.

File: README.md [hunk 1]
```diff
@@ -1,239 +1,24 @@
...
 Code for training and benchmarking morphology appropriate representation learning methods.
 
-# Preprocessing
-This README gives instructions for running our analysis against preprocessed image data. Because the preprocessing can take a long time, we are publishing preprocessed data along with the original movies. To do the preprocessing yourself, see the instructions in [subpackages/image_preprocessing/README.md](./subpackages/image_preprocessing/README.md).
+Our analysis is organized as follows.
 
-# Installation
+1. Single cell images
+2. Preprocessing (result: pointclouds and SDFs)
+    1. Alignment, masking, and registration
```
Comment (ritvikvasan) [inline README.md:9]: caveat with preprocessing -  alignment, masking and registration only used for the image models for punctate structures. For polymorphic structures, image models rely on a voxelized SDF/seg image

**Review by ritvikvasan — APPROVED** [review]
Looks good mostly. I think for the workflow, we should split up the preprocessing steps between punctate and polymorphic models.

File: README.md (hunk 1 above)
Comment (pgarrison) [reply README.md]: Done in [3f5da87](https://github.com/AllenCell/benchmarking_representations/pull/54/commits/3f5da87097e55887e09f27ec81159bf2edc36d09)

**Pull request merged**